src/libraries/Extensions/Microsoft.Agents.Extensions.Teams/Compat/TeamsActivityHandler.cs

```diff
@@ -297,10 +297,10 @@ protected virtual async Task<MessagingExtensionActionResponse> OnTeamsMessagingE
                 switch (action.BotMessagePreviewAction)
                 {
                     case "edit":
-                        return await OnTeamsMessagingExtensionBotMessagePreviewEditAsync(turnContext, action, cancellationToken).ConfigureAwait(false);
+                        return await OnTeamsMessagingExtensiOnAgentMessagePreviewEditAsync(turnContext, action, cancellationToken).ConfigureAwait(false);
 
                     case "send":
-                        return await OnTeamsMessagingExtensionBotMessagePreviewSendAsync(turnContext, action, cancellationToken).ConfigureAwait(false);
+                        return await OnTeamsMessagingExtensiOnAgentMessagePreviewSendAsync(turnContext, action, cancellationToken).ConfigureAwait(false);
```

Comment:
seems like a typo here

OnTeamsMessagingExtensiOnAgentMessagePreviewSendAsync
OnTeamsMessagingExtensionOnAgentMessagePreviewSendAsync

Reply:
got caught up in the rename of OnBot